reposync/env.sh.template

```diff
@@ -0,0 +1,3 @@
+export PUBLIC_REPO_ADDR=1.1.1.1
```

Comment:
is it correct address?

Reply:
Nope, this is just a template file with example configuration. There's an env.sh file with correct, used values on ci-repo. Consult the documentation on TCP repo sync here: https://review.opencontrail.org/#/c/51283/9/source/admin/repos-registries.rst (section TCP sync).